tests/auto/tst_dbus_interface.qml

```diff
@@ -187,7 +187,7 @@ TestCase {
         repr, {type:'g',value:"tussi"},           "repr: signature",             'signature:"tussi"',
         echo, {type:'g',value:"tussi"},           "echo: signature",             "tussi",
         repr, {type:'h',value:2},                 "repr: fd",                    /^fd:[1-9][0-9]*$/,
-        echo, {type:'h',value:2},                 "echo: fd",                    /^[1-9][0-9]*$/,
+        echo, {type:'h',value:2},                 "echo: fd",                    "undefined (json)",    // value is not demarshalled
```

Comment:
@adenexter Does this change look okay? From what I can tell in the tests, since the fd is not demarshalled, then the echoed value is not extractable.

Reply:
You could probably just exclude it from the test set as the file descriptor type doesn't translate to QML. It's just some opaque object in a QVariant.

Reply:
Yeah that makes more sense. Removed.